_includes/social/utterances.html

```diff
@@ -0,0 +1,7 @@
+<script src="https://utteranc.es/client.js"
+        repo="sylhare/Type-on-Strap"
+        issue-term="pathname"
+        theme="dark"

```

Comment:
You should use Liquid variables as you created them at `_config`.
Eg. `theme="dark"` should be `theme="{{ site.utterances.theme }}"`